src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_ICoalesceOperation.cs

```diff
@@ -1229,5 +1229,81 @@ void F(object alternative, object result)
 ";
             VerifyFlowGraphForTest<BlockSyntax>(compilation, expectedGraph);
         }
+
+        [CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)]
+        [Fact]
+        public void CoalesceOperation_14()
```

Comment:
No VB version?

Reply:
@AlekseyTs suggested covering one language for missing well known members is fine, especially given these are highly unlikely in real scenarios.

Reply:
> No VB version?

We just need to see that the node we produce in this situation makes sense, the node doesn't depend on specific language, therefore, I think that covering just for one language is sufficient. Especially given our time constraints.

---
In reply to: [190438966](https://github.com/dotnet/roslyn/pull/27082#discussion_r190438966) [](ancestors = 190438966)